dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgDataModel.ts

```diff
@@ -28,54 +28,48 @@ export const lksgDataModel = [
               "Is your head office, administrative headquarters, registered office or subsidiary located in Germany?",
             label: "Head Office in Germany",
             component: "YesNoFormField",
-            required: true,
-            validation: "required",
+            required: false,
           },
           {
             showIf: (): boolean => true,
             name: "groupOfCompanies",
             description: "Do you belong to a group of companies?",
             label: "Group of Companies",
             component: "YesNoFormField",
-            required: true,
-            validation: "required",
+            required: false,
           },
           {
             showIf: (dataModel: LksgData): boolean => dataModel?.general?.masterData?.groupOfCompanies === "Yes",
```

Comment:
Furthermore, I don't really like the name `showIf`.
It really confused me when I read it for the first time.
Some alternative name suggestions:

`isSubcategoryDisplayedInForm()`
`isDisplayedInForm()`
`isDisplayed()`
`isSubcategoryRendered()`
...

Reply:
I do not share that concern. Maybe we can discuss this briefly.

Reply:
Ok we can talk about it.
Another point I don't like about it, is that it makes you think that it is just hidden if `showIf()` returns `false`, while in fact it is not rendered at all in this case.

Reply:
Agreed on not renaming